recipes/pyqg/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pyqg" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 719d7634da033b675431983dde1615f2af702ecf62f61e118584692cfd70210d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pyfftw
+    - cython
```

Comment:
...but cython is definitely in the build environment!

Reply:
It should be in host. I know, not really the best choice of names there.